packages/components/src/__alpha__/SingleSelect/SingleSelect.spec.tsx

```diff
@@ -16,11 +17,90 @@ const SingleSelectWrapper = (): JSX.Element => (
 )
 
 describe('<SingleSelect />', () => {
+  afterEach(cleanup)
   describe('renders', () => {
     it('a basic select component', () => {
       const { getByRole } = render(<SingleSelectWrapper />)
       const select = getByRole('button')
       expect(select).toBeInTheDocument()
     })
   })
+
+  describe('popover interactions', () => {
```

Comment:
I keen to get thoughts about potentially moving the interaction tests over to Storybook's play tests 🤔 

I don't think we have a hard and fast rule about when to use one over the other so might also be worth discussing as a team when we're all back :)